app/views/correspondenceAddress.scala.html

```diff
@@ -36,8 +36,7 @@
 @implicitFormInstance = @{ Some(correspondenceAddressForm) }
 
 @pageScripts = {
-
-    <script type="text/javascript" src='@routes.Assets.versioned("javascript/location-autocomplete.min.js")'></script>
+    <script type="text/javascript" src='@routes.Assets.versioned("../assets/javascript/accessible-autocomplete.min.js")'></script>
```

Comment:
I don't believe this is what should have happened - we are using the new location-autocomplete as part of the new lookup for countries done under DL-8903

Reply:
Resolved - original implementation returned